### ISSUE TITLE
A fatal error has been detected by the Java Runtime Environment

### ISSUE THREAD

**Issue by rbakhtaraev** [bell-sw/Liberica#14]
Hi, i use image bellsoft/liberica-openjdk-alpine with tag "11". After latest update (11.0.6-10) to jdk 11.0.6+10-LTS and alpine 3.11, i started to get errors:

```
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGILL (0x4) at pc=0x00007f41513ed7f2, pid=109, tid=110
#
# JRE version:  (11.0.6+10) (build )
# Java VM: OpenJDK 64-Bit Server VM (11.0.6+10-LTS, mixed mode, tiered, compressed oops, g1 gc, linux-amd64)
# Problematic frame:
# v  ~StubRoutines::call_stub
#
```

system info: 
```
CPU:total 8 (initial active 8) (8 cores per cpu, 1 threads per core) family 6 model 85 stepping 4, cmov, cx8, fxsr, mmx, sse, sse2, sse3, tsc
CPU Model and flags from /proc/cpuinfo:
model name      : Intel(R) Xeon(R) Gold 6132 CPU @ 2.60GHz
flags           : fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ss ht syscall nx lm constant_tsc rep_good nopl xtopology eagerfpu pni cx16 hypervisor lahf_lm ssbd ibrs ibpb spec_ctrl flush_l1d

Memory: 4k page, physical 7972648k(1955304k free), swap 0k(0k free)

vm_info: OpenJDK 64-Bit Server VM (11.0.6+10-LTS) for linux-amd64 JRE (11.0.6+10-LTS), built on Jan 15 2020 01:25:34 by "bellsoft" with gcc 7.3.1 20180303 (Red Hat 7.3.1-5)
```

Stack:

```
Stack: [0x00007f416b8b5000,0x00007f416b9b6000],  sp=0x00007f416b9b4560,  free space=1021k
Native frames: (J=compiled Java code, A=aot compiled Java code, j=interpreted, Vv=VM code, C=native code)
v  ~StubRoutines::call_stub
V  [libjvm.so+0x887149]  JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, Thread*)+0x3b9
V  [libjvm.so+0x85851e]  InstanceKlass::call_class_initializer(Thread*)+0x22e
V  [libjvm.so+0x858a38]  InstanceKlass::initialize_impl(Thread*)+0x3c8
V  [libjvm.so+0x8588d6]  InstanceKlass::initialize_impl(Thread*)+0x266
V  [libjvm.so+0xdc0316]  Threads::initialize_java_lang_classes(JavaThread*, Thread*)+0x76
V  [libjvm.so+0xdc1303]  Threads::create_vm(JavaVMInitArgs*, bool*)+0x503
V  [libjvm.so+0x90e7b1]  JNI_CreateJavaVM+0x51
C  [libjli.so+0x4ac6]  JavaMain+0x86
```

On other CPUs it is not observed.

Rollback to version 11.0.5-11 solves problem.

**Comment by voitylov** [comment]
Providing instructions how to reproduce the environment would help ensure the fix is proper. If this is some VM config, it would be particularly interesting how a Skylake processor ends up reporting no AVX extensions.

**Comment by voitylov** [comment]
Fixed in the upcoming 11.0.7 release. Meanwhile, for configs where AVX is disabled on Skylake manually or via KVM/Hyper-V settings, use -XX:UseAVX=0 as a workaround.
